checkstyle/checkstyle.xml

```diff
@@ -112,7 +112,9 @@
     </module>
 
     <!-- code quality -->
-    <module name="MethodLength"/>
+    <module name="MethodLength">
+      <property name="max" value="170" />
+    </module>
```

Comment:
Because `AbstractRequest.doParseRequest()` and `AbstractResponse.parseResponse()` now exceed 150 lines.